test/test.tf

```diff
@@ -0,0 +1,72 @@
+variable "dns_zone" {
+  default = "Z2J3KVPABDNIL1"
+}
+
+provider "aws" {
+  version                 = "1.55"
+  region                  = "ap-southeast-2"
+  alias                   = "ap-southeast-2"
+  shared_credentials_file = "/aws/credentials"
+  profile                 = "aeternity"
+}
+
+variable "vault_addr" {
+  description = "Vault server URL address"
+}
+
+variable "bootstrap_version" {
+  default = "v2.0.1"
```

Comment:
I prefer master, this way the module maintainers are forced to keep it up-to-date with master